src/Entity/CitizenProjectCommitteeSupport.php

```diff
@@ -85,6 +85,11 @@ public function getCommittee(): Committee
         return $this->committee;
     }
 
+    public function getStatus(): string
```

Comment:
You don't have to add the getter if it's not used.

Reply:
This getter is used by Sonata in the Citizen Project list view.